go.mod

```diff
@@ -1,25 +1,26 @@
 module volcano.sh/volcano-global
 
-go 1.22.0
+go 1.22.9
 
 require (
-	github.com/karmada-io/karmada v1.10.2
+	github.com/karmada-io/karmada v0.0.0-00010101000000-000000000000
 	github.com/spf13/pflag v1.0.5
 	gomodules.xyz/jsonpatch/v2 v2.4.0
 	k8s.io/api v0.30.2
 	k8s.io/apimachinery v0.30.2
+	k8s.io/apiserver v0.30.2
 	k8s.io/client-go v0.30.2
 	k8s.io/component-base v0.30.2
-	k8s.io/klog/v2 v2.120.1
+	k8s.io/klog/v2 v2.130.1
 	k8s.io/kube-openapi v0.0.0-20240620174524-b456828f718b
-	sigs.k8s.io/controller-runtime v0.17.5
+	sigs.k8s.io/controller-runtime v0.18.4
 	volcano.sh/apis v1.10.0
 	volcano.sh/volcano v1.10.0
 )
 
 // todo: remove this when resourceBinding suspend merge to master
-// PR ref: https://github.com/karmada-io/karmada/commit/3058bbf50b953d02166625e2b797cb16c7f26396
-replace github.com/karmada-io/karmada => github.com/Vacant2333/karmada v0.0.0-20240606155424-3058bbf50b95
+// Ref: https://github.com/Vacant2333/karmada/tree/suspend-rb-2
+replace github.com/karmada-io/karmada => github.com/Vacant2333/karmada v0.0.0-20241207135009-0a9b0e40a707
```

Comment:
change to the correct branch, we will remove this when karmada supports resource binding suspension